Maquetado/scss/tech.scss

```diff
@@ -0,0 +1,45 @@
+.tech {
+  background-color: $green;
+  border-radius: 5px;
+  padding: 48px;
+  position: relative;
+  width: 250px;
+  
+  &:before {
+    content: "";
+    border-radius: 5px;
+    border:1px solid $green;
+    height:100%;
+    left: -10px;
+    position: absolute;
+    top: -10px;
+    width: 100%;
+    z-index: -1;
+  }
+  
+  .img-tech {
+    height: $tech-logo-size;
+    margin: 0 0 10px 0;
+    width: 100%;
+  }
+  
+  .title-tech{
+    display: block;
+    font-size: $text-big;
+    margin: 0 0 10px 0;
+    text-align: center;
+  }
+  
+  .release-tech {
+    display: block;
+    font-weight: bold;
+    font-size: $text-medium;
+    text-align: center;
+  }
+  
+  .txt-tech{
+    display: block;
+    padding: 20px 0 0 0;
```

Comment:
this can be done by using a margin bottom of 20px in the `.release-tech` instead